### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by agardnerIT** [keptn-sandbox/keptn-lifecycle-toolkit-docs#4]
This page should include text to clarify the difference between the lifecycle toolkit and "mainline Keptn".

**Comment by StackScribe** [comment]
I totally agree, but I would say it should show differences AND similarities.  At a high level, the two products have a lot of similar goals even if implemented differently.
